BackendIntegrationTests/BackendIntegrationTests.swift

```diff
@@ -236,34 +236,34 @@ class BackendIntegrationTests: XCTestCase {
         try AvailabilityChecks.iOS15APIAvailableOrSkipTest()
         configurePurchases()
         
-        var maybeProductID: String?
+        var productID: String?
         var completionCalled = false
-        var maybeReceivedEligibility: [String: IntroEligibility]?
+        var receivedEligibility: [String: IntroEligibility]?
         
         Purchases.shared.getOfferings { offerings, error in
-            maybeProductID = offerings?.current?.monthly?.storeProduct.productIdentifier
+            productID = offerings?.current?.monthly?.storeProduct.productIdentifier
             completionCalled = true
         }
         
         expect(completionCalled).toEventually(beTrue(), timeout: .seconds(10))
         completionCalled = false
         
-        let productID = try XCTUnwrap(maybeProductID)
+        let unwrappedProductID = try XCTUnwrap(productID)
```

Comment:
Ditto in all these. I think you can reuse the name.

Reply:
I get an `invalid redeclaration` when trying to reuse the name in these
<img width="883" alt="image" src="https://user-images.githubusercontent.com/3922667/151798464-89b1771a-036d-4374-8a25-0aabe21ba739.png">

Reply:
@NachoSoto I'm merging these for now, but happy to follow up with another PR if we come up with a good way to handle them

Reply:
Oh this is fine then!